setup.cfg

```diff
@@ -12,6 +12,7 @@ python-tag = py3
 
 [pydocstyle]
 match = template_filters.py
+match = flow/labels.py
```

Comment:
Please verify whether this overrides the previous "match" or if it checks both files. I'm not sure what behavior to expect from specifying it twice. In signac we used a complicated regular expression to match all desired filenames, examples below.

Initially we started with a list of allowed (files):
https://github.com/glotzerlab/signac/blob/e8209473103f562fe89c54bea1f5d1ba82b9632a/setup.cfg#L17-L19

Then later, once most of the code was compliant, we transitioned to a list of disallowed / skipped files:
https://github.com/glotzerlab/signac/blob/5186bb27621a1707b9d7493e14e551fe6426737d/setup.cfg#L17-L25